<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompteDialog</class>
 <widget class="QDialog" name="AddCompteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="typel">
      <property name="text">
       <string>Virtuel ou réel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Virtuel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Réel</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Type du compte</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox2">
      <item>
       <property name="text">
        <string>Actif</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Passif</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Depense</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Recette</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="noml">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditA"/>
    </item>
    <item>
     <widget class="QLabel" name="soldel">
      <property name="text">
       <string>Solde</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditB"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddCompteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCompteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
